utils/src/main/scala/cromwell/pipeline/utils/configs/LogInfoUtils.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.pipeline.utils.configs
+
+import cromwell.pipeline.utils._
+import cromwell.pipeline.utils.configs.SecretData._
+import play.api.libs.functional.syntax.{ toFunctionalBuilderOps, unlift }
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json.{ Json, Writes, __ }
+
+trait LogInfoUtils {
+  def getConfigMessage: String
+}
+
+object LogInfoUtils {
+
+  def getConfigMessage(applicationConfig: ApplicationConfig): String = {
```

Comment:
From my side, method's name is confusing. We can come up with the name corresponding to the action performed by the method.
How about `configToJson` or `configToJsonString`?

Reply:
Fixed